index.bs

```diff
@@ -560,76 +586,535 @@ message? -->
 
-When [[PRESENTATION-API|Presentation API]] [section
-6.7.1](https://www.w3.org/TR/presentation-api/#monitoring-incoming-presentation-connections)
+When [[PRESENTATION-API#monitoring-incoming-presentation-connections|section
+6.7.1]]
 says "it MUST listen to and accept incoming connection requests from a
 controlling browsing context using an implementation specific
 mechanism", the [=receiving user agent=] must receive and process the
 presentation-connection-open-request.
 
-When [[PRESENTATION-API|Presentation API]] [section
-6.7.1](https://www.w3.org/TR/presentation-api/#monitoring-incoming-presentation-connections)
-says "Establish the connection between the controlling and receiving browsing
-contexts using an implementation specific mechanism.", the [=receiving user
-agent=], must send a presentation-connection-open-response message.
+When [[PRESENTATION-API#monitoring-incoming-presentation-connections|section
+6.7.1]] says "Establish the connection between the controlling and receiving
+browsing contexts using an implementation specific mechanism.", the [=receiving
+user agent=], must send a presentation-connection-open-response message.
 
 
-Remote Playback API Protocol {#remote-playback}
------------------------------------------------
+Remote Playback Protocol {#remote-playback-protocol}
+----------------------------------------------------
 
-Issue(12): Propose control protocol for Remote
-Playback API.
+This section defines the use of the Open Screen Protocol for starting, terminating,
+and controlling remote playback of media as defined by the
+[[REMOTE-PLAYBACK|Remote Playback API]].  [[#remote-playback-api]] defines how
+APIs in [[REMOTE-PLAYBACK|Remote Playback API]] map to the protocol messages
+defined in this section.  
+ 
+For all messages defined in this section, see Appendix A for the full
+CDDL definitions.
+
+<!-- TODO: Add a capability that indicates support for the
+remote playback protocol.
+See https://github.com/webscreens/openscreenprotocol/issues/123 -->
+<!-- TODO: Frequency to update current-time (HtmlMediaElement does every 250ms?) -->
+<!-- TODO: extended mime types in source tag? -->
+<!-- TODO: media queries replacing URLs? -->
+
+To learn which receivers are [=compatible remote playback device=]s (also called
+available [=remote playback devices=]) for a particular URL or set of URLs, the
+controller may send a remote-playback-availability-request message with the
+following values:
+
+: urls
+:: A list of [=media resources=].  Must not be empty.
+
+: headers 
+:: headers that the receiver should use to fetch the
+    urls.  For example,
+    [[REMOTE-PLAYBACK#establishing-a-connection-with-a-remote-playback-device|section 6.2.4 of
+    the Remote Playback API]] says that the Accept-Language header should be
+    provided.
+
+: watch-duration
+:: The period of time that the controller is interested in receiving updates
+    about the URLs, should the availability change.
+
+: watch-id
```

Comment:
That's a good question.  If it's chosen by the receiver (included in the response), then the receiver can easily control the uniqueness (like we do with presentation connection IDs).  But currently it's chosen by the controller, which means that the receiver must keep a table of [receiver, watch_id] in a table instead of just [watch_id].  But maybe that's OK.  

For now, I'll just say it MUST be unique across watches of a given type to the same receiver.   Do you really think we need it to be unique across types?

Reply:
I'll make a github issue to discuss.